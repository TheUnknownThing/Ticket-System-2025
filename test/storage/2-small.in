239
insert 15 467522174
insert 21 286350929
insert 29 987538117
insert 25 1105837348
delete 21 286350929
insert 15 1952183979
insert 9 986798949
insert 24 1687008100
insert 17 1431110084
insert 25 352453281
insert 1 590428826
insert 9 1648582593
find 29
find 9
delete 17 1431110084
insert 15 1393078180
insert 30 1821960223
delete 15 1952183979
insert 7 972443421
insert 13 1386955932
delete 29 987538117
insert 11 853379183
find 7
delete 13 1386955932
insert 3 504364009
insert 18 1829652243
find 9
find 11
insert 20 1308957191
insert 1 1120444507
insert 26 1794865937
insert 6 495973057
insert 16 208455342
delete 1 590428826
delete 7 972443421
insert 26 878856398
insert 26 1710456454
delete 6 495973057
find 25
insert 16 532615851
insert 22 976092708
delete 22 976092708
insert 18 990034927
insert 20 1573258158
insert 24 1249429525
delete 25 1105837348
insert 26 1178062274
insert 12 342117792
insert 10 193757177
insert 25 560846881
insert 26 1853600129
insert 14 367856706
insert 27 1620724978
insert 17 1862911463
delete 26 1794865937
insert 24 1324532077
insert 20 278645838
insert 24 1751336771
delete 3 504364009
insert 6 313533941
insert 29 98476963
find 25
insert 17 1163861793
insert 21 302985989
insert 12 960995102
delete 26 1853600129
insert 8 32229561
insert 6 81424606
insert 8 950703324
insert 12 833236008
insert 15 832198806
insert 4 760205555
insert 28 1664623750
delete 21 302985989
insert 5 870476914
insert 29 52985593
insert 29 1517983136
insert 7 437500951
delete 25 560846881
insert 16 343884524
insert 25 1724562838
find 25
find 4
insert 3 1240975228
insert 20 1452793068
insert 26 122754600
insert 29 1418292974
insert 19 34565533
insert 20 1866896437
insert 30 263949102
delete 1 1120444507
delete 20 1573258158
delete 10 193757177
insert 10 1781852491
delete 17 1862911463
insert 11 533260402
insert 4 2082988501
insert 6 1068937371
insert 28 947152174
insert 9 1282766225
insert 26 443970378
insert 29 323954313
insert 21 149520769
delete 20 1308957191
insert 5 851549195
insert 20 1909720016
delete 9 986798949
insert 4 631224718
delete 18 990034927
insert 13 1003745933
insert 17 1605625620
delete 20 1866896437
insert 3 927181579
insert 20 1760671866
insert 11 916149569
insert 16 163001638
insert 30 1976102243
insert 15 955466202
delete 13 1003745933
insert 9 1323808033
insert 4 1657185797
insert 4 1953492518
insert 21 1707505865
delete 7 437500951
insert 17 1468380467
insert 5 1889483363
insert 1 1069768977
insert 20 60865534
insert 2 1266242378
insert 30 1293409953
insert 26 575412853
insert 28 1022948855
insert 29 671967460
insert 18 1294474073
find 3
insert 26 672592083
insert 30 45056774
insert 8 1340383324
delete 20 60865534
insert 20 541340633
insert 14 1990284209
insert 17 594265846
delete 28 1022948855
insert 5 1195577075
delete 3 927181579
insert 14 1692369308
insert 14 1205196214
delete 16 532615851
delete 11 916149569
find 29
find 26
delete 5 851549195
find 26
insert 22 1909674284
insert 11 1839415932
insert 9 2130452841
insert 8 620602865
insert 16 865290266
insert 29 830509592
insert 5 706828368
insert 6 1894786191
insert 20 2140971226
insert 2 1841281726
find 9
insert 28 1009392010
insert 2 1434317054
insert 13 914572848
insert 2 1150857387
insert 22 635055987
delete 10 1781852491
insert 5 536468855
insert 29 2077668362
insert 2 245353288
insert 26 662493042
insert 20 1499159681
insert 26 1930724277
insert 26 1932077263
insert 17 1110496471
insert 28 600948675
insert 22 115744866
insert 30 1699065254
insert 20 1494293935
insert 7 1970806358
insert 30 344382983
insert 1 1356907031
insert 1 1993480677
find 20
insert 2 340892812
find 14
insert 24 225947956
insert 25 1268785384
insert 18 1244123364
find 18
insert 22 75664119
insert 22 2144446066
insert 25 1397514739
insert 17 1667346347
insert 13 775867454
delete 28 1664623750
insert 25 538903919
insert 5 344618634
insert 23 1945881969
insert 13 38925529
insert 21 1880526009
delete 9 1648582593
insert 18 309097938
insert 4 303466837
insert 11 2097526659
insert 28 327490106
insert 10 1227078993
insert 3 1729343327
insert 1 1742336147
insert 3 486260917
insert 16 183818822
delete 23 1945881969
insert 11 1184600268
find 17
insert 27 653455473
insert 23 1437212839
insert 4 752867123
insert 8 1910307923
insert 16 739530696
insert 30 1357105993
insert 18 2090413039
insert 27 737137761
insert 13 939767776
insert 16 131200241
insert 11 1869653038
insert 15 1236908532
delete 25 538903919
insert 29 1899963521
delete 1 1069768977
delete 29 1418292974
find 28
insert 2 1415634628
insert 27 254321501
insert 15 2056031999
insert 13 2119066355
find 17