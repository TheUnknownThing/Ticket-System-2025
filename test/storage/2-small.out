987538117 
986798949 1648582593 
972443421 
986798949 1648582593 
853379183 
352453281 1105837348 
352453281 560846881 
352453281 1724562838 
760205555 
927181579 1240975228 
52985593 98476963 323954313 671967460 1418292974 1517983136 
122754600 443970378 575412853 672592083 878856398 1178062274 1710456454 
122754600 443970378 575412853 672592083 878856398 1178062274 1710456454 
1282766225 1323808033 1648582593 2130452841 
278645838 541340633 1452793068 1494293935 1499159681 1760671866 1909720016 2140971226 
367856706 1205196214 1692369308 1990284209 
1244123364 1294474073 1829652243 
594265846 1110496471 1163861793 1468380467 1605625620 1667346347 
327490106 600948675 947152174 1009392010 
594265846 1110496471 1163861793 1468380467 1605625620 1667346347 
